src/app/services/error.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class ErrorService {
+
+  constructor() { }
+
+   handleError (error: Response | any) {
+    // In a real world app, we might use a remote logging infrastructure
```

Comment:
Well, its almost a real-world app isn't it 😏 ? jk ... we should definitely log errors remotely in the future.